src/api.ts

```diff
@@ -16,10 +16,20 @@ function get<T>(url: string): Promise<T> {
   // eslint-disable-next-line prefer-template
   const fullURL = BASE_URL + url + '.json';
 
-  // we add some delay to see now the laoder works
+  // we add some delay to see now the loader works
   return wait(300)
     .then(() => fetch(fullURL))
-    .then(res => res.json());
+    .then(response => {
+      if (!response.ok) {
+        return Promise.reject(new Error('Server does not work'));
```

Comment:
Consider rephrasing the error message to provide more context: 'Failed to fetch data from server'